submissions/EmmanuelUdoh/phpTask/overview.php

```diff
@@ -0,0 +1,42 @@
+<?php
+require "helper.php";
+require "config.php";
+
+try {
+    $connection = new PDO("mysql:host=$host", $username, $password, $options);
+    $sql = file_get_contents("../data/fetch.sql");
+    $stmt = $connection->query("SELECT * FROM kimaj.products");
+    $products = array();
+    while ($item = $stmt->fetch()) {
+        array_push($products, $item);
+    }
+    $_SESSION['products'] = $products;
```

Comment:
You can remove `$_SESSION['products'] = $products;` you are using the `$products array` you are populating in lines 10 and 11.